src/IssueViz.Security/Taint/TaintIssueToIssueVisualizationConverter.cs

```diff
@@ -27,12 +27,15 @@
 using SonarLint.VisualStudio.IssueVisualization.Models;
 using SonarLint.VisualStudio.IssueVisualization.Security.Taint.Models;
 using SonarQube.Client.Models;
+using SonarQube.Client.Models.ServerSentEvents.ClientContract;
+using ITaintIssue = SonarQube.Client.Models.ServerSentEvents.ClientContract.ITaintIssue;
 
 namespace SonarLint.VisualStudio.IssueVisualization.Security.Taint
 {
     interface ITaintIssueToIssueVisualizationConverter
     {
         IAnalysisIssueVisualization Convert(SonarQubeIssue sonarQubeIssue);
+        IAnalysisIssueVisualization Convert(ITaintIssue sonarQubeTaintIssue);
```

Comment:
I figured it would be more appropriate that the existing converter handles the new format as well.